.babelrc

```diff
@@ -1,50 +1,88 @@
 {
   "env": {
-    "test": {
+    "development": {
       "presets": [
-        ["env", {"targets": { "node": "current" }}],
+        [
+          "env",
+          {
+            "targets": {
+              "browsers": ["last 2 versions", "ie >= 9"]
+            }
+          }
+        ],
         "react",
         "stage-2"
-      ],
-      "plugins": [
-        "babel-plugin-rewire",
-        ["module-resolver", {
```

Comment:
Why do we remove this from the `test` env?

Reply:
That's why the tests were failing, thank you!